components/org.wso2.carbon.identity.user.endpoint/src/gen/java/org/wso2/carbon/identity/user/endpoint/dto/SuccessfulUserCreationDTO.java

```diff
@@ -1,95 +1,86 @@
+    /**
+    * Status code of the operation.
+    **/
+    @ApiModelProperty(value = "Status code of the operation.")
+    @JsonProperty("code")
+    public String getCode() {
+        return code;
+    }
+    public void setCode(String code) {
+        this.code = code;
+    }
 
-  
-  /**
-   * Account confirmation notification sent channel.
-   **/
-  @ApiModelProperty(value = "Account confirmation notification sent channel.")
-  @JsonProperty("notificationChannel")
-  public String getNotificationChannel() {
-    return notificationChannel;
-  }
-  public void setNotificationChannel(String notificationChannel) {
-    this.notificationChannel = notificationChannel;
-  }
+    /**
+    * Descriptive message regarding the operation.
+    **/
+    @ApiModelProperty(value = "Descriptive message regarding the operation.")
+    @JsonProperty("message")
+    public String getMessage() {
+        return message;
+    }
+    public void setMessage(String message) {
+        this.message = message;
+    }
 
-  
-  /**
-   * Confirmation code to verify the user, when notifications are externally managed. In this scenario, *notificationChannel* will be *EXTERNAL*. Use the confirmation code for confirm the user accounut.
-   **/
-  @ApiModelProperty(value = "Confirmation code to verify the user, when notifications are externally managed. In this scenario, *notificationChannel* will be *EXTERNAL*. Use the confirmation code for confirm the user accounut.")
-  @JsonProperty("confirmationCode")
-  public String getConfirmationCode() {
-    return confirmationCode;
-  }
-  public void setConfirmationCode(String confirmationCode) {
-    this.confirmationCode = confirmationCode;
-  }
+    /**
+    * Account confirmation notification sent channel.
+    **/
+    @ApiModelProperty(value = "Account confirmation notification sent channel.")
+    @JsonProperty("notificationChannel")
+    public String getNotificationChannel() {
+        return notificationChannel;
+    }
+    public void setNotificationChannel(String notificationChannel) {
+        this.notificationChannel = notificationChannel;
+    }
 
-  
+    @Override
+    public String toString() {
 
-  @Override
-  public String toString()  {
-    StringBuilder sb = new StringBuilder();
-    sb.append("class SuccessfulUserCreationDTO {\n");
-    
-    sb.append("  code: ").append(code).append("\n");
-    sb.append("  message: ").append(message).append("\n");
-    sb.append("  notificationChannel: ").append(notificationChannel).append("\n");
-    sb.append("  confirmationCode: ").append(confirmationCode).append("\n");
-    sb.append("}\n");
-    return sb.toString();
-  }
+        StringBuilder sb = new StringBuilder();
+        sb.append("class SuccessfulUserCreationDTO {\n");
+        
+        sb.append("    code: ").append(code).append("\n");
+        sb.append("    message: ").append(message).append("\n");
+        sb.append("    notificationChannel: ").append(notificationChannel).append("\n");
+        
```

Comment:
Shall we remove this new line, unwanted break

Reply:
These are auto-generated classes. It's better if we did not change them.